### ISSUE TITLE
 Error: Cannot find module 'ng2-google-place-autocomplete'

### ISSUE THREAD

**Issue by albertorealfdez** [psykolm22/ng2-google-place-autocomplete#2]
Hi again,

Some days ago some things on the application's architecture I am using were made, and now I am not able to import this module. I installed it several times, resolved the unmet peer dependencies, but I still get the error `Error: Cannot find module 'ng2-google-place-autocomplete'`

I have tried different things, like importing the module via `node_modules/ng2-google-place-autocomplete/` folder, and that error disappears, but I got a different one: `Uncaught Error: Unexpected value 'GooglePlaceModule' imported by the module ...`. I got this error before, but it seemed to be an installation problem.

I do not know what the error can be caused by. 

Thank you very much in advance.

Regards,

Alberto.

**Comment by psykolm22** [comment]
Hi,

Did you create your project with angular-cli ? 
Did you use webpack config ?

**Comment by albertorealfdez** [comment]
Sorry I didn't specify again. No, the project is build with Angular Universal and Webpack .

**Comment by psykolm22** [comment]
Ok i have reproduce this config and i have the same problem. i'm trying to find solution .

**Comment by psykolm22** [comment]
i think it come from dependencies version . Can you show me your package.json dependencies ?

**Comment by albertorealfdez** [comment]
Oh, good. Let's see, this is the original `package.json`:

```
"dependencies": {
    "@angular/common": "2.1.0",
    "@angular/compiler": "2.1.0",
    "@angular/core": "2.1.0",
    "@angular/forms": "2.1.0",
    "@angular/http": "2.1.0",
    "@angular/platform-browser": "2.1.0",
    "@angular/platform-browser-dynamic": "2.1.0",
    "@angular/platform-server": "2.1.0",
    "@angular/router": "3.1.0",
    "@types/ua-parser-js": "0.7.30",
    "angular2-express-engine": "~2.1.0-rc.1",
    "angular2-platform-node": "~2.1.0-rc.1",
    "angular2-recaptcha": "^0.3.2",
    "angular2-universal": "~2.1.0-rc.1",
    "angular2-universal-polyfills": "~2.1.0-rc.1",
    "body-parser": "1.15.2",
    "bootstrap-sass": "3.3.7",
    "express": "4.14.0",
    "fs-extra": "0.30.0",
    "methods": "~1.1.2",
    "ng2-bootstrap": "^1.1.14",
    "ng2-google-place-autocomplete": "1.2.9",
    "ng2-social-share": "0.0.10",
    "ng2-webstorage": "1.3.1",
    "preboot": "4.5.2",
    "rxjs": "5.0.0-beta.12",
    "ua-parser-js": "0.7.10",
    "zone.js": "0.6.21"
  },
  "devDependencies": {
    "@angularclass/resolve-angular-routes": "1.0.9",
    "@types/body-parser": "0.0.29",
    "@types/compression": "0.0.29",
    "@types/cookie-parser": "1.3.29",
    "@types/express": "4.0.32",
    "@types/express-serve-static-core": "4.0.33",
    "@types/hammerjs": "2.0.32",
    "@types/mime": "0.0.28",
    "@types/node": "6.0.38",
    "@types/serve-static": "1.7.27",
    "angular2-template-loader": "0.4.0",
    "awesome-typescript-loader": "2.2.4",
    "cookie-parser": "1.4.3",
    "imports-loader": "0.6.5",
    "json-loader": "0.5.4",
    "css-loader": "0.25.0",
    "extract-text-webpack-plugin": "2.0.0-beta.4",
    "file-loader": "0.9.0",
    "jasmine-core": "2.5.2",
    "karma": "1.3.0",
    "karma-jasmine": "1.0.2",
    "karma-junit-reporter": "^1.1.0",
    "karma-phantomjs-launcher": "1.0.2",
    "karma-sourcemap-loader": "0.3.7",
    "karma-webpack": "1.8.0",
    "node-inspector": "0.12.8",
    "node-sass": "3.8.0",
    "nodemon": "1.10.0",
    "raw-loader": "0.5.1",
    "string-replace-loader": "github:gdi2290/string-replace-loader",
    "rimraf": "2.5.4",
    "sass-loader": "4.0.2",
    "style-loader": "0.13.1",
    "ts-loader": "0.8.2",
    "ts-node": "1.3.0",
    "typescript": "2.0.2",
    "typings": "1.3.3",
    "url-loader": "0.5.7",
    "wdio-mocha-framework": "0.4.3",
    "wdio-phantomjs-service": "0.1.0",
    "wdio-spec-reporter": "0.0.3",
    "wdio-testingbot-service": "0.1.6",
    "webdriverio": "4.2.15",
    "webpack": "2.1.0-beta.25",
    "webpack-dev-middleware": "1.6.1",
    "webpack-dev-server": "2.1.0-beta.9",
    "webpack-merge": "0.13.0"
  }
```

However, I tried what you said, and installed the new versions of typescript and zone.js libraries, since I thought the same than you and there were two  UNMET PEER DEPENDENCY errors, and I still got the same problem.

Thank you.

**Comment by psykolm22** [comment]
Have you some UNMET PEER DEPENDENCY error message when npm install ng2-google-place-autocomplete ?

**Comment by albertorealfdez** [comment]
Exactly, I had two UNMET PEER DEPENDENCY errors with the two libraries I mentioned before, but I solved installing the last version of them. I explained it at the end of my previous comment, in case you did not see it.

**Comment by psykolm22** [comment]
I have update dependency and clean js . Hope version 1.3.2 will solve it . i have make test with a fresh angular2-weckpack-starter  project and it work.

**Comment by albertorealfdez** [comment]
Now I have a different `UNMET PEER DEPENDENCY rxjs@5.0.0-rc2` when installing the 1.3.2 version of the library, and if I run the application I have the same problem of `Cannot find module 'ng2-google-place-autocomplete'`. However, in my package.json I have exactly that version of rxjs library.

**Comment by psykolm22** [comment]
In your last post with your package.json file it's "rxjs": "5.0.0-beta.12", are you sure that your update to rxjs@5.0.0-rc2 ? I think ng2-google-place-autocomplete refuse to install if you don't have rxjs@5.0.0-rc2 .

**Comment by albertorealfdez** [comment]
Sorry, I didn't explain it right. I meant to say that I updated the rxjs library to that version, so in my `package.json` file it's "rxjs": "5.0.0-rc.2".

**Comment by psykolm22** [comment]
Can i clone your project to understand what is going on ?

**Comment by albertorealfdez** [comment]
I am sorry, but it is a private repository, and it is not in my hand. 

Anyway,  I tried again with some architecture changes and now I don't get any UNMET PEER DEPENDENCY error, but I still have the same problem when running the application: `Error: Cannot find module 'ng2-google-place-autocomplete'`

**Comment by psykolm22** [comment]
Should be fix now with Release 1.4.1
